apps/site/assets/css/_utilities.scss

```diff
@@ -163,9 +163,9 @@
       z-index: 1;
     }
   }
+}
 
-  a:not(&__primary-link) {
-    position: relative;
-    z-index: 2;
-  }
+.u-linked-card a:not(&__primary-link) {
```

Comment:
Probably need to add `// scss-lint:disable` here

Reply:
updated